.circleci/config.yml

```diff
@@ -1,7 +1,76 @@
-version: 2
+version: '2.1'
```

Comment:
What is the main distinction between 2 and 2.1?

Reply:
@DylanBaker bump